Source/WebKit/WebProcess/GPU/media/RemoteVideoFrameProxy.cpp

```diff
@@ -55,28 +55,28 @@ RemoteVideoFrameProxy::Properties RemoteVideoFrameProxy::properties(WebKit::Remo
     };
 }
 
-Ref<RemoteVideoFrameProxy> RemoteVideoFrameProxy::create(IPC::Connection& connection, RemoteVideoFrameObjectHeapProxy& videoFrameObjectHeapProxy, Properties&& properties)
+Ref<RemoteVideoFrameProxy> RemoteVideoFrameProxy::create(Ref<IPC::Connection> connection, Ref<RemoteVideoFrameObjectHeapProxy> videoFrameObjectHeapProxy, Properties&& properties)
```

Comment:
These two changes seem unnecessary. Generally, we don't need to change the function arguments from raw reference/pointer to Ref/RefPtr.

Reply:
My idea was to make sure the caller would do the right thing implicitly, and in case of `create` functions and constructors there's no extra ref-count churn because the `Ref` gets moved into the object.
But as discussed, you've explained that the overall strategy is that callers are responsible for keeping function arguments alive (it's more consistent everywhere), so I'll redo this patch accordingly...